packages/server/graphql/mutations/addComment.ts

```diff
@@ -69,6 +72,23 @@ const addComment = {
     const {id: commentId, isAnonymous, threadParentId} = dbComment
     await r.table('Comment').insert(dbComment).run()
 
+    if (discussion.discussionTopicType === 'teamPromptResponse') {
```

Comment:
at some point, it'd be great to have these notifications also for sprint poker and other types of meetings but that's a different task for sure

Reply:
I think this notification is pretty specific to standups, since it's "notify whoever this full discussion thread's stage is relevant to", which doesn't really correlate with any other meetings, apart from maybe notifying whoever added an issue on sprint poker